features/product/viewing_products/viewing_products_form_specific_taxon.feature

```diff
@@ -0,0 +1,33 @@
+@viewing_products
+Feature: Viewing products from specific taxon
+    In order to see products from specific category
+    As a Visitor
+    I want to be able to view products from selected category
+
+    Background:
+        Given the store operates on a channel named "Poland"
+        And the store operates on another channel named "France"
+        And the store classifies its products as "T-Shirts" and "Funny"
+        And the store has product "T-Shirt Banana" which is available in "Poland" channel
+        And this product belongs to "T-Shirts"
+        And this product belongs to "Funny"
+        And the store has product "Plastic Tomato" which is available in "Poland" channel
+        And this product belongs to "Funny"
+        And the store has product "T-Shirt Batman" which is available in "France" channel
+        And this product belongs to "T-Shirts"
+
+    @todo
+    Scenario: Viewing products from specific taxon
+        Given I want to see products from specific category
+        When I change my current channel to "Poland"
+        And I select "T-Shirts" category
+        Then I should see "T-Shirt Banana"
+        And I should not see "T-Shirt Batman"
+        And I should not see "Plastic Tomato"
+
+    @todo
+    Scenario: Viewing information about empty list of products from given category
+        Given I want to see products from specific category
```

Comment:
Scenario: Viewing information about empty list of products from **a** given category
Given I want to see products from **a** specific category